### PR TITLE
Rewording placeholders and validating vesting cliff is lesser than vesting period

### PR DESCRIPTION
**Pull request by Peartes**
This commit address 3 minor issues [here](https://www.notion.so/curvelabs/339a249185e14ddcbd731e5ba07c46c7?v=f7cba822bb9043a5a87ab80b5e2b40ef&p=a3cae64b0b474447bb226d32a9893a3e) [here](https://www.notion.so/curvelabs/339a249185e14ddcbd731e5ba07c46c7?v=f7cba822bb9043a5a87ab80b5e2b40ef&p=f2bb5b8ed038413f852b2271d62d0217) and [here](https://www.notion.so/curvelabs/339a249185e14ddcbd731e5ba07c46c7?v=f7cba822bb9043a5a87ab80b5e2b40ef&p=f2bb5b8ed038413f852b2271d62d0217)

### CODE REVIEW

File: src/newSeed/stage2.html [hunk 1]
```diff
@@ -61,7 +61,7 @@
       <div class="infoInput logo">
         <div>
           <label for="logo">Project Logo</label>
-          <input id="logo" type="text" placeholder="Valid Uri to project's logo"
+          <input id="logo" type="text" placeholder="Please enter a valid URL to project's logo"
```
Comment (dkent600) [inline src/newSeed/stage2.html:64]: I don't agree with this change.  Placeholders are always rounds, not requests.  The wording you are proposing is appropriate for validation messages.

A better wording: "Url to the project's logo"  (note that Url is more familiar to the public than Uri)

File: src/newSeed/stage5.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
       <div>
         <div class="infoInput">
           <label for="email">Contact Email</label>
-          <input id="email" type="email" placeholder="Valid email address"
+          <input id="email" type="email" placeholder="Please enter a valid email address"
```
Comment (dkent600) [inline src/newSeed/stage5.html:11]: I don't agree with this change.  Placeholders are always rounds, not requests.  The wording you are proposing is appropriate for validation messages.

A better wording: "Project email address"

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/newSeed/stage2.html (hunk 1 above)
Comment (Peartes) [reply src/newSeed/stage2.html:64]: Okay, I will do that. @BartuCulha please note this change is different from what you requested in the cards

File: src/newSeed/stage5.html (hunk 2 above)
Comment (Peartes) [reply src/newSeed/stage5.html:11]: Okay well @BartuCulha please note this also, thanks

**Review by dkent600 — APPROVED** [review]

**Pull request merged**